src/condor_tools/htcondor_cli/annex_create.py

```diff
@@ -22,8 +23,102 @@
 REMOTE_MKDIR_TIMEOUT = int(htcondor.param.get("ANNEX_REMOTE_MKDIR_TIMEOUT", 5))
 REMOTE_POPULATE_TIMEOUT = int(htcondor.param.get("ANNEX_REMOTE_POPULATE_TIMEOUT", 60))
 
-# FIXME: Make this come from config? At least some place not hardcoded.
-MACHINE_QUEUE_MAP = {"stampede2": "normal"}
+#
+# For queues with the whole-node allocation policy, cores and RAM -per-node
+# are only informative at the moment; we could compute the necessary number
+# of nodes for the user (as long as we echo it back to them), but let's not
+# do that for now and just tell the user to size their requests with the
+# units appropriate to the queue.
+#
+MACHINE_TABLE = {
```

Comment:
Seems like we should store this in an external JSON file or something, not important now though.

Reply:
Agreed, added to the long list of "other" things we need to to do.